docs/source/native/functions.rst

```diff
@@ -35,6 +35,14 @@ Conditional operators can also be used to make comparisons, such as ``==`` (equa
   
 Like other languages, if statements can be connected to ``else if`` and ``else`` statements. ``else if`` statements must be used immediately after an ``if`` or ``else if`` and will only check their condition if the preceding statements failed. ``else`` statements behave similarly, but always run if the preceding statements failed and must be last.
 
+You can also use short if statements to simplify your code. However, if not used carefully this can worsen readability.
+The Syntax is ``condition ? if_condition_true : if_condition_false``. This is especially useful when declaring variables or passing parameters.
+
+.. code-block:: javascript
+
+  // shortenedUsername is "longus..."" if username is "longusername" or "short" if username is "short"
+  string shortenedUsername = username.len() > 9 ? username.slice(0,6) + "..." : username;
+
```

Comment:
This is called a ternary operation, might want to mention that, since it's a pattern that is shared across a good number of languages